com.alovoa.alovoa-electron.yml

```diff
@@ -0,0 +1,81 @@
+app-id: com.alovoa.alovoa-electron
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-run/keyring
+  # required for notifications in various desktop environments
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Tray
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=org.ayatana.indicator.application
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.*
+  # File system access
+  - --filesystem=home
+  - --filesystem=xdg-data/themes
+  - --filesystem=xdg-config/gtk-3.0
+  - --filesystem=xdg-config/gtk-4.0
+  - --filesystem=xdg-config/kdeglobals:ro
```

Comment:
or even this permission...

Reply:
That's for detecting light/dark system themes on GNOME and KDE